football_explorer/explorer.py

```diff
@@ -2,13 +2,24 @@
 
 from .models import Player
 
-
-class FootballExplorer(object):
+class FootballExplorer():
     def __init__(self, csv_file_name):
         self.csv_file_name = csv_file_name
 
     def all(self):
-        raise NotImplementedError()
+        with open('test_data.csv') as csvfile:
+            player = csv.reader(csvfile, delimiter=',', quotechar='|')
+            return [Player(row[0],row[1],row[2],row[3],row[4],row[5],row[6],row[7],row[8]) for row in player]
```

Comment:
you can use the shorthand to pass all parameters like so:-

`return [Player(*row) for row in player]`

To understand what * means, you can check out https://www.programiz.com/python-programming/args-and-kwargs#args

Since this is an assignment on iterator, you can also turn this into a generator! 

```python
for row in player:
    yield Player(*row)
```